tests/conftest.py

```diff
@@ -29,3 +32,14 @@ def run(self, command, **kwargs):
                 raise Exception('Unsupported command in mock: {}'.format(' '.join(command)))
 
     return MockDocker()
+
+
+@pytest.fixture
+def hello_archive(tmpdir):
```

Comment:
Nit: you could use `tmp_path`, which is the newer version of the temporary directory fixture